.github/README.md

```diff
@@ -779,8 +779,16 @@ Click the button below, to deploy to Vercel 👇
 
 ### Deploying - Option #3: Docker
 
+Pull Image: `docker pull lissy93/web-check`
```

Comment:
> Pull Image: `docker pull lissy93/web-check`

I don't think this is necessary, as if the image isn't found locally, it'll be pulled automatically during the run command. The exception being, unless they user wants to use a different registry (like GHCR). But in this instance, running that pull command will just download it from DockerHub, same as in the run command.

Reply:
Okay cool. You're right!